charts/sn-platform/templates/vault/vault-statefulset.yaml

```diff
@@ -32,9 +32,22 @@ spec:
   serviceAccount: {{ template "pulsar.vault.serviceAccount" . }}
   serviceType: {{ .Values.vault.serviceType }}
   statsdDisabled: true
+  {{- if .Values.vault.securityContext }}
+  securityContext: {{- toYaml .Values.vault.securityContext | nindent 4 }}
+  vaultEnvsConfig:
+    - name: SKIP_CHOWN
+      value: "true"
+    - name: SKIP_SETCAP
+      value: "true"
+    - name: VAULT_DISABLE_MLOCK
```

Comment:
Disabling mlock is not recommended: https://www.vaultproject.io/docs/configuration#disable_mlock

Reply:
emmm, seems `mlock` needs root privilege. I got error message when I try to run vault as non-root

```
Error initializing core: Failed to lock memory: cannot allocate memory

This usually means that the mlock syscall is not available.
Vault uses mlock to prevent memory from being swapped to
disk. This requires root privileges as well as a machine
that supports mlock. Please enable mlock on your system or
disable Vault from using it. To disable Vault from using it,
set the `disable_mlock` configuration option in your configuration
file.
2022-08-11T01:26:29.465Z [INFO]  proxy environment: http_proxy="" https_proxy="" no_proxy=""
```